### ISSUE TITLE
[BUG] Error migrating pre-v0.6.0 dataset to v0.6.2

### ISSUE THREAD

**Issue by brimoor** [voxel51/fiftyone#607]
I tried to load a persistent dataset that I had created with a pre-video version of FiftyOne using the latest develop, and experienced the following error:

The dataset is an image dataset with content that looks like the screenshot posted below:

```py
import fiftyone as fo

dataset = fo.load_dataset("bdd100k-validation")
```

```
Migrating dataset 'bdd100k-validation' to the current version (0.6.2)
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-2-ae96572947cd> in <module>
----> 1 dataset = fo.load_dataset("bdd100k-validation")

~/dev/fiftyone/fiftyone/core/dataset.py in load_dataset(name)
     88         ValueError: if no dataset exists with the given name
     89     """
---> 90     return Dataset(name, _create=False)
     91 
     92 

~/dev/fiftyone/fiftyone/core/singleton.py in __call__(cls, name, _create, *args, **kwargs)
     34         ):
     35             instance = cls.__new__(cls)
---> 36             instance.__init__(name=name, _create=_create, *args, **kwargs)
     37             name = instance.name  # `__init__` may have changed `name`
     38             cls._instances[name] = instance

~/dev/fiftyone/fiftyone/core/dataset.py in __init__(self, name, persistent, _create)
    201                 self._sample_doc_cls,
    202                 self._frame_doc_cls,
--> 203             ) = _load_dataset(name)
    204 
    205         self._deleted = False

~/dev/fiftyone/fiftyone/core/dataset.py in _load_dataset(name)
   2051                 % (dataset_doc.name, VERSION)
   2052             )
-> 2053             runner.run(dataset_names=[dataset_doc.name])
   2054         dataset_doc.reload()
   2055         dataset_doc.version = VERSION

~/dev/fiftyone/fiftyone/migrations/runner.py in run(self, dataset_names)
     45             for revision, module in self._revisions_to_run:
     46                 fcn = etau.get_function(self.direction, module)
---> 47                 fcn(conn, dataset_name)
     48 
     49     @property

~/dev/fiftyone/fiftyone/migrations/revisions/v0_6_2.py in up(db, dataset_name)
     22         dataset_dict["media_type"] = "image"
     23 
---> 24     for field in dataset_dict["sample_fields"] + dataset_dict["frame_fields"]:
     25         if "media_type" in field:
     26             del field["media_type"]

KeyError: 'frame_fields'
```

<img width="1655" alt="fiftyone-v0 6 1" src="https://user-images.githubusercontent.com/25985824/96067996-e5386c80-0e68-11eb-9abe-60eaebd4d4f6.png">

**Comment by brimoor** [comment]
@benjaminpkane Also, I can't seem to reproduce this at the moment, but I've also seen a bug where I get an error when **exiting** my Python process with a dataset loaded. The error is a missing `version` key somewhere.

I will try to track this down in more detail. I think it was on `develop` with https://github.com/voxel51/fiftyone/pull/576 merged.

**Comment by benjaminpkane** [comment]
Looking into it

**Comment by benjaminpkane** [comment]
> @benjaminpkane Also, I can't seem to reproduce this at the moment, but I've also seen a bug where I get an error when **exiting** my Python process with a dataset loaded. The error is a missing `version` key somewhere.
> 
> I will try to track this down in more detail. I think it was on `develop` with #576 merged.

This is likely from using a `v0.6.2.` dataset in a prior version of fiftyone (pre-migrations). I don't think there is much we can do about that. Or it is at least not worth the engineering, imo.

**Comment by benjaminpkane** [comment]
Resolved by #601
